src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1912,6 +1912,16 @@ static bool processSubscriptions
 
         if (cSubP != NULL)
         {
+          if (cSubP->status == STATUS_ONESHOT)
+          {
+            BSONObj query  = BSON("_id" << OID(mapSubId));
+            BSONObj update = BSON("$set" << BSON(CSUB_STATUS << STATUS_INACTIVE));
+            // update the status to inactive as status is oneshot
```

Comment:
Suggestion:

// update the status to inactive as status is oneshot -> // update the status to inactive as status is oneshot (in both DB and csubs cache)

To make the comment more complete

Reply:
fixed in acb7d41